### PR TITLE
Add platform comparison page

### PR DESCRIPTION
**Pull request by dlech**

### CODE REVIEW

**Comment by WasabiFan** [comment]
Can we get some HTML formatting?

**Comment by dlech** [comment]
There is some at the bottom. :grin:

**Comment by dlech** [comment]
And there is a preview: http://dlech.github.io/ev3dev.github.io/docs/platform-comparison/

**Comment by WasabiFan** [comment]
😵 that's a fancy table. That might be the most complicated table I've ever seen.

> There is some at the bottom. 😁

Is there something preventing an auto-formatter from formatting it?

**Comment by dlech** [comment]
I tried using a fancy html editor, but I guess it was not that fancy. should be better now.

**Comment by WasabiFan** [comment]
Ah, that's much more readable!

What's happening with the multi-column cells here (and in a few other places)? Is it just because there were two next to each other which said the same thing? I would prefer it if they were separate cells.

image

Could you try making the border on the first column more visible to differentiate the headings from the values?

I think this page might benefit from a sentence of intro/explanation.

**Comment by dlech** [comment]
> What's happening with the multi-column cells here (and in a few other places)? Is it just because there were two next to each other which said the same thing?

Yes

> I would prefer it if they were separate cells.

I *knew* you were going to say that.

> Could you try making the border on the first column more visible to differentiate the headings from the values?

Any css tips for this?

**Comment by dlech** [comment]
I also need to add a link from the main docs page.

**Comment by WasabiFan** [comment]
> I knew you were going to say that.

I'm not actually OCD, but I've found myself to be a bit of a perfectionist :laughing:

> Any css tips for this?

Something like this, probably:

```css
.table-index-column {
    border-right-color: white;
    border-right-width: 3px;
}
```

Then you'd apply that class to each cell in the column. There may be a Bootstrap class for this already, but I doubt it; you'll probably just have to fiddle with the color and weight.

**Comment by WasabiFan** [comment]
You may want to wrap this monolithic table in a `<div class="table-responsive">` to make it scrollable on mobile.

**Comment by dlech** [comment]
:new:

File: docs/platform-comparison.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
+---
+title: Platform Comparison
+---
+
+<p>
+    This table provides a detailed comparison of all of the ev3dev supported
```
Comment (WasabiFan) [inline docs/platform-comparison.html]: Hyphenate "ev3dev-supported"

File: docs/platform-comparison.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    devices and their features.
+</p>
+
+<div class="table-responsive">
+    <table class="table table-striped table-bordered table-left-header">
```
Comment (WasabiFan) [inline docs/platform-comparison.html]: Nice thinking with the header class

File: docs/platform-comparison.html [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+            <td>LEGO</td>
+            <td colspan="2">Dexter Industries</td>
+            <td>mindsensors.com</td>
+            <td>FatcatLab</td>
+            <td>Quest Institute</td>
+        </tr>
+        <tr>
+            <th colspan="2">Website</th>
+            <td><a href="http://mindstorms.lego.com" target="_blank">mindstorms.lego.com</a></td>
+            <td colspan="2"><a href="http://www.dexterindustries.com/BrickPi" target="_blank">dexterindustries.com</a></td>
```
Comment (WasabiFan) [inline docs/platform-comparison.html]: What's this "target" stuff?

Comment (dlech) [inline docs/platform-comparison.html]: open in a new page

**Comment by dlech** [comment]
:new: :new:  http://dlech.github.io/ev3dev.github.io/docs/platform-comparison/

**Comment by dlech** [comment]
I tried `.lead` and it looked out of place with no regular text afterwards, so leaving as-is for now.

**Pull request merged**